forge-gui/res/cardsfolder/upcoming/green_slime.txt

```diff
@@ -0,0 +1,10 @@
+Name:Green Slime
+ManaCost:2 G
+Types:Creature Ooze
+PT:2/2
+K:Flash
+K:Foretell G
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Any | Destination$ Battlefield | Execute$ TrigCounterAbility | SpellDescription$ When CARDNAME enters the battlefield, counter target activated or triggered ability from an artifact or enchantment source. If a permanent's ability is countered this way, destroy that permanent.
+SVar:TrigCounterAbility:DB$ Counter | TargetType$ Activated,Triggered | TgtPrompt$ Select target activated or triggered ability from an artifact or enchantment source. | ValidTgts$ Artifact,Enchantment | DestroyPermanent$ True
+AI:RemoveDeck:All
```

Comment:
@Agetian should this be All or Random?

Reply:
@squee1968 – unless @Agetian or someone advises otherwise, just change this to `RemoveDeck:Random`

To the best of my understanding:
`RemoveDeck:All` means the AI really just can't play it for whatever reason
`RemoveDeck:Random` means that the AI shouldn't add it to decks when generating or drafting because it is a dud card without proper support cards or situations

Reply:
Yep, I can confirm that. Looks correct on this one.